cmd/ready/config.go

```diff
@@ -23,7 +23,6 @@ func MustNewConfig(v *viper.Viper) *Config {
 			mapstructure.StringToTimeDurationHookFunc(),
 			mapstructure.StringToTimeHookFunc(time.RFC3339),
 			mapstructure.StringToSliceHookFunc(","),
-			hook.StringToFeatureHookFunc(),
```

Comment:
Did you intentionally delete the featureHook here?

Reply:
fixed